src/modules/menu/components/search/SearchBar.vue

```diff
@@ -91,3 +91,12 @@ export default {
     },
 }
 </script>
+
+<style lang="scss" scoped>
+
+.form-control {
+    font-size: 0.825rem;
+    font-style: italic;
+}
```

Comment:
I don't think the current viewer has italic font, does it?
Also, it might be better to use a custom class here instead of overwriting things on a bootstrap class, it's all fine because you scoped the style, but as soon as someone will remove this `scoped` part (because he might want to do something globally) then your rule will be applied to all inputs styled with bootstrap.